osp_checks.py

```diff
@@ -10,94 +10,66 @@
 from utils import ssh_client
 
 
-def check_vm_count(system, warn=10, crit=15, **kwargs):
-    """ Check overall vm count. """
-    logger = kwargs["logger"]
+def check_threshold(count, warn, crit, logger):
```

Comment:
This is a good idea, can you please pass `resource` as a `str` variable, in which we can pass things like `vm,keypair,..] and so on` when we call the function?

Reply:
you do not need to explicitly mentioned the data type here, python takes data type dynamically, any way for safety measure i have convert these values to integer, in case some one passed it as string.